src/pages/join/volunteering.js

```diff
@@ -20,6 +21,53 @@ const TitleHeading = styled.h1`
   `}
 `;
 
+const MyBtn = styled.button`
+    background-color: rgba(34,34,34,1);
+    border-color: rgba(34,34,34,1);
+    color: rgba(238,238,238,1);
+    width: 30%;
+    line-height: 1.5;
+    font-size: 1.2vw;
+    border-radius: 8px;
+    margin-bottom: 2rem;
+    cursor: pointer;
+
+    &:hover {
+      background-color: rgba(50,50,50,1);
+    }
+
+    @media (max-width: 768px) {
+      width: 80%;
+      font-size: 4vw;
+    }
+`;
+
+const MyBtn2 = styled.button`
+    background-color: rgba(238,238,238,1);
+    border-color: rgba(238,238,238,1);
+    color: rgba(34,34,34,1);
+    width: 30%;
+    line-height: 1.5;
+    font-size: 1.2vw;
+    border-radius: 8px;
+    margin-bottom: 2rem;
+    cursor: pointer;
+
+    &:hover {
+      background-color: rgba(220,220,220,1);
+    }
+
+    @media (max-width: 768px) {
+      width: 80%;
+      font-size: 4vw;
+    }
+`;
+
+const btns = {
+  light: <MyBtn>Apply for a volunteer position</MyBtn>,
+  dark: <MyBtn2>Apply for a volunteer position</MyBtn2>
+};
+
 
 const TopicHeading = ({ children }) => <b>{`${children} · `}</b>;
```

Comment:
let changes this to "Apply here", or simply "Apply". the text ends up on two lines depending on the screen size